packages/documentation/docs/controls/buttons/_button_code.md

```diff
@@ -75,6 +75,34 @@ name="button-grey-ghost"
 hideInitalCodePreview
 examplesByName></Playground>
 
+### Danger
```

Comment:
Maybe we could add here a `ApiTableSinceTag` https://github.com/siemens/ix/blob/dfbc082d073873a4ae5902ed0e3ecdfe4133708f/packages/documentation/src/components/ApiTableTag/index.tsx